encoding/_yaml/type/binary.ts

```diff
@@ -117,6 +117,9 @@ function representYamlBinary(object: Uint8Array): string {
 }
 
 function isBinary(obj: Any): obj is Buffer {
+  if (typeof obj !== "object" || !obj || obj.readSync === undefined) {
```

Comment:
Should the `readSync` part be `readFromSync`? If so, could you also change this to `typeof obj.readFromSync !== 'function'`

Reply:
Oh 🤦 You're completely right, I apologize. I made the fix, though `"function"` instead of `'function'` for consistency with the other strings in the file.